packages/dapper/src/dapper.ts

```diff
@@ -87,10 +105,26 @@ export class Dapper implements DapperInterface {
     return transform(cleanedData);
   }
 
+  protected async dummyAndProcess<Schema extends z.ZodTypeAny, ReturnType>(
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    dummyFunc: (props: any) => ReturnType,
+    queryParams: QsArray,
+    transform: (cleanedData: z.infer<Schema>) => ReturnType
+  ): Promise<ReturnType> {
+    const cleanedData = await dummyFunc(queryParams);
+    return transform(cleanedData);
+  }
+
   /**
    * Methods implementing the DapperInterface.
    */
-  getCommunityPackageListing = getCommunityPackageListing;
-  getFrontpage = getFrontpage;
+  getPackageListings = getPackageListings;
+  getCommunities = getCommunities;
+  getCommunity = getCommunity;
   getPackage = getPackage;
+  getTeam = getTeam;
+  getUser = getUser;
+  getOldCommunityPackageListing = getOldCommunityPackageListing;
```

Comment:
Maybe move `getOld*` at the bottom and add a comment stating they're deprecated, intended for the old nextjs app and will be removed eventually?